ivy_tests/test_ivy/test_functional/test_nn/test_layers.py

```diff
@@ -51,7 +51,7 @@ def x_and_linear(draw, dtypes):
     dtype_x_weight_bias=x_and_linear(
         dtypes=helpers.get_dtypes("numeric", full=False),
     ),
-    ground_truth_backend="jax",
+    test_with_out=st.just(False),
```

Comment:
We shouldn't be setting `test_with_out=False` as it wouldn't run tests with the `out` argument